cody.hh

```diff
@@ -5,7 +5,12 @@
 #ifndef CODY_HH
 #define CODY_HH 1
 
-#include "cody-conf.h"
+#if defined(__has_include) && __has_include("cody-conf.h")
+# include "cody-conf.h"
+#else
+# include "cody-conf.h"
```

Comment:
you're including in both arms of the if.  That seems unintended?

Reply:
yeah, that's not what I intended...

I meant
```
#if defined(__has_include)
#  if __has_include("cody-conf.h")
#   include "cody-conf.h"
# endif
#else
#   include "cody-conf.h"
#endif
```

to cover the case that the compiler doesn't support __has_include by falling back to having to install it even if empty.  Perhaps there's no C++11 compiler now that fail to support __has_include ?